frontend/src/components/Navbar/Navbar.js

```diff
@@ -17,47 +17,43 @@ export default function Navbar() {
           to="/"
           data-testid="home-link"
         >
-          <HomeIconStyled />
+          <HomeIcon alt="Link zur Homepage" />
         </LinkStyled>
         <LinkStyled
           activeClassName="selected"
           to="/statements"
           data-testid="statements-link"
         >
-          <StatementIconStyled alt="Statements" />
+          <StatementIcon alt="Link zur Selbsteinschätzung" />
         </LinkStyled>
         <LinkStyled
           activeClassName="selected"
           to="/result"
           data-testid="result-link"
         >
-          <ResultIconStyled alt="Result" />
+          <ResultIcon alt="Link zum Ergebnis" />
         </LinkStyled>
         <LinkStyled
           activeClassName="selected"
           to="/entries"
           data-testid="entries-link"
         >
-          <EditIconStyled alt="Entries" />
+          <EditIcon alt="Link zum Eingabeformular" />
         </LinkStyled>
         <LinkStyled
           activeClassName="selected"
           to="/topics"
           data-testid="topics-link"
         >
-          <TopicsIconStyled alt="Topics" />
+          <TopicsIcon alt="Link zur Themensammlung" />
         </LinkStyled>
       </NavbarStyled>
     </>
   )
 }
 
 const NavbarStyled = styled.footer`
-  position: fixed;
```

Comment:
Without position: fixed; do you think there might be some issues in certain browsers?

Reply:
I moved it into GlobalStyles, so it's still there, just easier to try to optimize the grid and position